src/templates/ira_doe_workflow_linear_fluorescent_to_LED_retrofit.mdx

```diff
@@ -0,0 +1,283 @@
+        Take a photo of the make and model number of the LED lamp.
+      </PhotoInput>
+      <PhotoInput id="led_lamp_details_photo_b" label="LED Lamp Color Temperature, Wattage, and Lumen – Photo">
+        Take a photo of the information about the color temperature, wattage, and lumen.
+      </PhotoInput>
+      <PhotoInput id="dimmer_make_model_photo_b" label="Dimmer Make and Model – Photo">
+        If a dimmer is used, take a photo of the make and model of the dimmer. Test the dimmer to make sure the light dimming works.
+      </PhotoInput>
+      <PhotoInput id="project_invoice_photo_b" label="Project Invoice – Photo">
+        Take a photo of the project invoice.
+      </PhotoInput>
+    </ShowOrHide>
+    <ShowOrHide visible={props.data?.led_tube_type === 'Type C'}>
+      <PhotoInput id="fixture_rewired_photo_c" label="Fixture Rewired – Photo">
+        Take a photo of the rewired fixture. If there are more than one fixture being retrofitted, take a representative photo.
+      </PhotoInput>
+      <PhotoInput id="led_driver_photo_c" label="LED Driver – Photo">
+        Take a photo of the LED driver. If there is a label or markings detailing the date of manufacturing, model number, maximum output current, etc., please photograph from close enough to read the information. This photo will be very helpful for maintenance staff in the future for ordering compatible replacement lamps.
+      </PhotoInput>
+      <PhotoInput id="led_lamp_make_model_photo_c" label="LED Lamp Make and Model – Photo">
+        Take a photo of the make and model number of the LED lamp.
+      </PhotoInput>
+      <PhotoInput id="led_lamp_details_photo_c" label="LED Lamp Color Temperature, Wattage, and Lumen – Photo">
+        Take a photo of the information about the color temperature, wattage, and lumen.
+      </PhotoInput>
+      <PhotoInput id="dimmer_make_model_photo_c" label="Dimmer Make and Model – Photo">
+        If a dimmer is used, take a photo of the make and model of the dimmer. Test the dimmer to make sure the light dimming works.
+      </PhotoInput>
+      <PhotoInput id="project_invoice_photo_c" label="Project Invoice – Photo">
+        Take a photo of the project invoice.
+      </PhotoInput>
+    </ShowOrHide>
+
+</Tab>
+
+  <Tab eventKey="report"  title="Report">
+    <PrintSection label="Print Report">
+    ---
+    # Linear Fluorescent to LED Retrofit
+    ## Photo Report
+
+    <ProjectInfoReport {...props} />
+
+    ## About the Report
+
+    This report is intended to document the linear fluorescent to LED retrofit work.
+
+    For the client/customer, this report contains information about the type of LED used
+    in this project as well as photos showing the make and model number the LED tubes installed.
+    This information is useful for when the time comes to purchase replacement LED tubes.
+    There are three ways to retrofit a fixture to LED, Type A, Type B, and Type C.
```

Comment:
Grammar. Replace with "There are three ways to retrofit a fixture to LED**:** Type A, Type B, and Type C."